### ISSUE TITLE
Spatial Harvesters needs blacklisted ores and mobs

### ISSUE THREAD

**Issue by Vynlore01** [Chuunkyz/BenchmarkII-1.18#59]
Using the Ore Harvester from Spatial Harvester gives you every ore it can find. This means getting multiple disabled versions of lead, tin, iridium, etc. ores that don't have proper recipes like the ones that generate naturally.
These are the ores I've noticed (also with deepslate variants):
For Modern Industrialization: Tin , Lead , Iridium , Baxuite, Salt, Tungsten
For Industrial Revolution: Tin, Lead, Silver, Tungsten
For Bewitchment: Salt, Silver
As well as the "Unobtainable in Survival Mode" (Deepslate) ores from Tech Reborn: Peridot, Sheldonite, Tungsten, Sodalite
I may have (?) missed a couple but these are the ones I've noticed.

It's also outputting BYG ores (idk if these are enabled or they are just rare-just saying cause I don't think I've seen any while mining.)

While a couple of these don't have any recipes, some (basically any Tin, Lead, Silver, Tungsten) actually do work in Industrial Revolution's Fluid Infuser to make purified versions of ores (because it uses ore tags). 

There's also the issue of the Dark Mob Harvester collecting boss drops from the Bosses of Mass Destruction mod. The Specific Mob Harvester may also be able to do this although I have not tested it.

~The configuration file has the ability to blacklist these ores/mobs/mods

TLDR: Ore Harvester from Spatial Harvester gives disabled ores; Dark Mob Harvester gives boss drops from BOMD.

**Comment by Vynlore01** [comment]
Forgot to mention, version: 0.6.8

**Comment by Chuunkyz** [comment]
ye, i'm aware for the ores, i was trying to remove the ore tags to every unobtaible ore but a kubejs update broke my thing so this is the reason of why it's still not done now
also thanks for reporting bomd drops, i was not aware of that

**Comment by Vynlore01** [comment]
Latest version (0.6.10) with the changed config still does not work. At first I thought it was a bug with Spatial Harvesters in that it didn't respect configuration choices, but as I changed the config in my test instance I noticed that it looks like you didn't get the format right in the config. (Although idk if you need the quotes, this is what worked however)

It should be like this:
[
        indrev,
        "lead_ore"
]

you had it like this:
[
        "indrev:lead_ore"
]

After changing each one to the correct format it worked, at least for me. The format should also be used for blacklisting the wither (or any singular blacklisted mob), as your current one does not work.
New:
[
        minecraft,
        wither
]

Old:
[
        "minecraft:wither"
]


Also, the values for betwitchment, byg and BOMD seem to work fine although I wasn't testing it THAT long.

**Comment by Chuunkyz** [comment]
ye, i figured this now
the config was not finished for 0.6.9 because i got crashes with custom config when making the thing, now i think i can make the whole thing functional
